go/felix/rules/static.go

```diff
@@ -429,6 +450,87 @@ func (r *DefaultRuleRenderer) StaticNATOutputChains(ipVersion uint8) []*Chain {
 	}}
 }
 
+func (r *DefaultRuleRenderer) StaticRawTableChains(ipVersion uint8) []*Chain {
+	return []*Chain{
+		r.failsafeInChain(),
+		r.failsafeOutChain(),
```

Comment:
Why do we put failsafes in the raw table?

I just checked that we did this in the 1.4 version too, but I can't remember why.  I'm thinking they are perhaps not needed there because we don't make any DROP decisions in the raw table.